website/source/docs/providers/google/r/compute_instance_template.html.markdown

```diff
@@ -136,6 +136,11 @@ The following arguments are supported:
 * `metadata` - (Optional) Metadata key/value pairs to make available from
     within instances created from this template.
 
+* `metadata_startup_script` - (Optional) An alternative to using the
+    startup-script metadata key, mostly to match the compute_instance resource.
+    This replaces the startup-script metadata key on the created instance and 
+    thus the two mechanisms are not allowed to be used simultaneously.
```

Comment:
> It would make `metadata_startup_script` incompatible with setting any metadata at all

Ah I thought that was true. Given:

```
  metadata {
    foo = "bar"
  }
  metadata_startup_script = "somescript"
```

Won't you get a perpetual diff in the metadata map as the two fields fight over whether or not `startup-script` should be in there?